src/marten/template/tag/for/loop.cr

```diff
@@ -36,6 +36,14 @@ module Marten
           def revindex0
             @items_size - @index - 1
           end
+
+          def odd?
+            !even?
+          end
+
+          def even?
+            @index % 2 == 0
+          end
```

Comment:
Why not use the existing `#odd?` and `#even?` methods?

```suggestion
          def odd?
            @index.odd?
          end

          def even?
            @index.even?
          end
```

Reply:
Whoa!! I haven't thought about it. Thanks @ellmetha. I'll make the change.